src/sentry/static/sentry/app/views/settings/components/settingsNavigation.tsx

```diff
@@ -44,14 +45,21 @@ class SettingsNavigation extends React.Component<Props> {
     const navWithHooks = navigationObjects.concat(hookConfigs);
 
     return (
-      <div>
+      <PositionStickyWrapper>
         {navWithHooks.map(config => (
           <SettingsNavigationGroup key={config.name} {...otherProps} {...config} />
         ))}
         {hooks.map((Hook, i) => React.cloneElement(Hook, {key: `hook-${i}`}))}
-      </div>
+      </PositionStickyWrapper>
     );
   }
 }
 
+const PositionStickyWrapper = styled('div')`
+  position: sticky;
+  top: 100px;
+  overflow: scroll;
```

Comment:
```suggestion
```
Getting rid of `overflow:scroll;` seems to fix the extra gutters I was seeing but then causes another problem on very tiny screens.

Reply:
Ah – so I didn’t see the scrollbar in your gif. Sadly we need `min-height` and `overflow: scroll` because if the browser window height is small then users still need to scroll through those items. I think I have an alternative solution though.